package.json

```diff
@@ -9,14 +9,23 @@
     "url": "https://github.com/potaorg/pota"
   },
   "type": "module",
-  "types": "./ts.pota.d.ts",
   "main": "./src/exports.js",
-  "module": "./src/exports.js",
+  "// Using 'typesVersions' here is the only way we could figure out how to get types working for imports of any subpath without any of the problems other approaches have when not using modeResolution:NodeNext (listed in https://stackoverflow.com/questions/77856692/how-to-publish-plain-jsjsdoc-library-for-typescript-consumers)": "",
+  "typesVersions": {
+    "*": {
+      "src/*": ["types/*"]
+    }
+  },
```

Comment:
For people not on `moduleResolution:NodeNext` yet, this will allows types to be found for any subpath (but the shortcuts defined in `exports` will not work in that case as those require `NodeNext`).